chsdi/lib/validation/mapservice.py

```diff
@@ -93,7 +95,18 @@ def chargeable(self):
     def where(self, value):
         ## TODO regexp to test validity of sql clause
         if value is not None:
-            self._where = value
+            if 'ilike' in value:
+                match = re.search(r'(.*)(%.*%)(.*)', value)
+                where = ''.join((
+                    match.group(1).replace('\'', 'E\''),
+                    match.group(2).replace('\'', '\\\''),
+                    match.group(3)
+                ))
+                where = where.encode('ascii', 'replace')
```

Comment:
Not really up to date here, but e.g. `ü` is a non-ascii character, but doesn't cause any problems?